src/main/java/wolox/training/constants/ErrorConstants.java

```diff
@@ -0,0 +1,7 @@
+package wolox.training.constants;
+
+public final class ErrorConstants {
+
+    public static final String nullMessage = "Please check the Object supplied, its null!";
```

Comment:
This message can be improved. The word ** object ** can be replaced with a placeholder which is then replaced with the name of the field using `String.format()`.